test/unit/object/test_item.py

```diff
@@ -105,3 +108,49 @@ def test_get(test_item_and_response, mock_box_session, fields, mock_object_id, e
     mock_box_session.get.assert_called_once_with(expected_url, params=expected_params, headers=if_none_match_header)
     assert isinstance(info, test_item.__class__)
     assert info.id == mock_object_id
+
+
+@pytest.mark.parametrize(
+    'size, name, file_id, parent_id, expected_url, expected_data',
+    [
+        (
+            # Testing preflight check of a new file to be uploaded to Box.
+            100,
+            'foo.txt',
```

Comment:
We are simply checking if we are making the call with the same name. But sure! :)